src/lib/core/decimal.c

```diff
@@ -30,8 +30,8 @@
  */
 
 #include "decimal.h"
-#include "third_party/decNumber/decContext.h"
-#include "third_party/decNumber/decPacked.h"
+#include <decNumber/decContext.h>
+#include <decNumber/decPacked.h>
```

Comment:
Shall we go deeper in this particular case and update include path to third_party/decNumber?